backend/views/syson-diagram-common-view/src/main/java/org/eclipse/syson/diagram/common/view/edges/TransitionEdgeDescriptionProvider.java

```diff
@@ -48,11 +57,50 @@ public abstract class AbstractTransitionEdgeDescriptionProvider extends Abstract
 
     private final IDescriptionNameGenerator descriptionNameGenerator;
 
-    public AbstractTransitionEdgeDescriptionProvider(IColorProvider colorProvider, IDescriptionNameGenerator descriptionNameGenerator) {
+    public TransitionEdgeDescriptionProvider(IColorProvider colorProvider, IDescriptionNameGenerator descriptionNameGenerator) {
         super(colorProvider);
         this.descriptionNameGenerator = Objects.requireNonNull(descriptionNameGenerator);
     }
 
+    private List<NodeDescription> getAllActionOrStateUsage(IViewDiagramElementFinder cache, boolean excudeStart, boolean excludeDone) {
```

Comment:
Please put private methods at the end of the file

Reply:
Done